frontend/src/app/modules/home/home-page/home-page.component.ts

```diff
@@ -27,13 +28,16 @@ export class HomeComponent extends BaseComponent implements OnInit, OnDestroy {
         public dashboardService: DashboardService,
         private updateDataService: ShareDataService<Dashboard>,
         private deleteDataService: ShareDataService<number>,
-        private toastNotificationService: ToastNotificationService
+        private toastNotificationService: ToastNotificationService,
+        private authService: AuthenticationService
     ) {
         super();
+        this.user = authService.getUser();
     }
 
     async ngOnInit() {
-        this.authorizedUser = { ...this.authorizedUser, firstName: 'Andriy' };
+        this.user = { ...this.user, firstName: this.user.firstName };
```

Comment:
Why do you need this? This line does nothing

Reply:
Delete this line